boldodod/boldodod.github.io
# 163053201 Responsive UI pages

#### What does this PR do?
Have responsive UI pages across all screen sizes

#### Description of Task to be completed?
Have the following pages
-Sign in page
-Signup Page
-Page for meetup topic with questions on a particular topic
-Admin page for creating and deleting meetups
-A profile page where the user can see the log of comments and questions

#### How should this be manually tested?
After cloning the repo, open index.html page on the browser and navigate through.
*use the browser developer options to check screen responsiveness.

#### Any background context you want to provide?


#### What are the relevant pivotal tracker stories?
163053201

#### Screenshots (if appropriate)
#### Questions:
